packages/ui/src/theme/tokens/components/table.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  /**
+   * Default borders
+   */
+  borderColor: { value: '{colors.border.tertiary.value}' },
+  borderStyle: { value: 'solid' },
+  borderWidth: { value: '{borderWidths.small.value}' },
+
+  /**
+   * Default padding
+   */
+  padding: { value: '{space.medium.value}' },
+
+  /**
+   * Default font styles
+   */
+  color: { value: '{colors.font.primary.value}' },
+  fontSize: {
+    default: { value: '{fontSizes.medium.value}' },
+    summary: { value: '{fontSizes.xs.value}' },
+  },
+  fontWeight: {
+    td: { value: '{fontWeights.normal.value}' },
```

Comment:
I think all the sub components should be properties at the top level.
```
{
  data: {
    fontWeight: {}
    }
  }
```
That way the CSS variables look like `--amplify-components-[primary]-[subcomponent]-font-weight` (`--amplify-components-table-data-font-weight`)

Reply:
Nice, much cleaner. I updated it to match this pattern and included individual tokens for each subcomponent.